File: 2-AD-HOC/3048-Sequencia_Secreta-1/3048.py
l=[]
for i in range(0,int(input())):
   l.append(int(input()))
bgl=l[0]
p=0
for i in l:
    if i ==bgl:
        p+=1
        if bgl==1:
            bgl=2
        else:
            bgl=1
print(p)

File: 2-AD-HOC/3048-Sequencia_Secreta-1/3048.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        int tamanho = int.Parse(Console.ReadLine());
        
        int[] l = new int [tamanho];
        for(int g=0; g<tamanho;g++){
            l[g]=int.Parse(Console.ReadLine());
        }
        int bgl = l[0];
        int p =0;
        foreach(int i in l){
            if(i==bgl){
                p++;
                if(bgl==1){
                    bgl=2;
                }
                else{
                    bgl=1;
                }
                
            }
        }
        Console.WriteLine(p);
    }   
}